extensions/amp-messaging-protocol/amp-messaging-protocol.md

```diff
@@ -0,0 +1,40 @@
+<!---
```

Comment:
Please rename this file to `*-internal.md`. That will keep it from showing up in user docs.